src/components/Sidebar.js

```diff
@@ -1,32 +1,44 @@
 import React, { Component } from 'react';
 import { Route, Link, Switch } from 'react-router-dom';
+import { withRouter } from 'react-router'
```

Comment:
good work on using `withRouter` from `react-router` to inject the history and other props from react router